### DISCUSSION TITLE
Pointer not showing Time on every cell in resourceTimelineDay

### DISCUSSION THREAD

**Discussion by SweAdeel** [vkurko/calendar#388] [Q&A]
Hello  !

Pointer not showing Time on every cell in resourceTimelineDay , i put pointer as true in calender initialization but not time not showing on cell 
and can we highlight the whole row once pointed to any cell ?
share some sample code if possible 

Thanks

**Comment by vkurko** [comment]
> Pointer not showing Time on every cell in resourceTimelineDay

Can you provide a live demo please? CodeSandbox, StackBlitz, etc
